frontend/src/shared/components/tasque-team-select/tasque-team-select.component.sass

```diff
@@ -0,0 +1,136 @@
+@import 'src/colors.sass'
+
+*
+  box-sizing: border-box
+  font-family: 'Nunito', sans-serif
+  font-style: normal
```

Comment:
No global styles in the component file... Move it to global.css

Reply:
Done